paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -648,6 +648,7 @@ const std::vector<std::string> kPirMkldnnPasses {
       "matmul_scale_fuse_pass",                   //
       "scale_matmul_fuse_pass",                   //
       "reshape_transpose_matmul_fuse_pass",       //
+      "matmul_reshape_add_pass",                  //
```

Comment:
Suggest move it after "matmul_add_act_fuse_pass" since "reshape_transpose_matmul_fuse_pass" and "matmul_transpose_reshape_fuse_pass" are serial.